src/views/Dashboards/Dashboards.js

```diff
@@ -82,8 +90,17 @@ const Dashboards = () => {
         </Card>
       </Grid>
       <Grid item sm={10} className={classes.dashboardsMainContainer}>
-        <Card className={classes.dashboard}>
-          <CardContent>{<TabPanel index={value} key={dashboardTitle} title={dashboardTitle} />}</CardContent>
+        <Card className={classes.dashboardsMainCard}>
+          <CardContent className={classes.dashboardsMainCardContent}>
+            {
```

Comment:
I think that brackets are useless here

Reply:
Fixed it, thank you :slightly_smiling_face: